src/home.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent } from 'react'
```

Comment:
The ts line annotation below from github action is wrong. It should be on line 8 instead of 1.

Reply:
did you figure it out how to fix it?